students/Hebdowska_Grazyna/lesson_06/uppercase.py

```diff
@@ -0,0 +1,19 @@
+def capitalize(lower_case_word):
+    new_word = []
+    for i in lower_case_word:
+        new_word.append(i)
+    new_word[0] = new_word[0].upper()
+    upper_case_word = ''
+    for i in new_word:
+        upper_case_word += i
```

Comment:
It's a bit complicated solution.
 
Please consider the following
```
title = lower_case_word[0].upper() + lower_case_word[1:]
```
or use dedicated method for this operation
```
lower_case_word.title()
```